unit_tests/library/tile.cpp

```diff
@@ -32,7 +32,7 @@ BOOST_AUTO_TEST_CASE(test_tile)
     const auto rc = osrm.Tile(params, result);
     BOOST_CHECK(rc == Status::Ok);
 
-    BOOST_CHECK_EQUAL(result.size(), 114033);
+    BOOST_CHECK_EQUAL(result.size(), 113824);
```

Comment:
I think it's because the duration values encoded in the vector tiles changed. It uses a delta encoding so the size is heavily data dependent.